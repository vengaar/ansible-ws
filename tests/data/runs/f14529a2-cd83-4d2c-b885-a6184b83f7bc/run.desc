{"playbook": "/home/vengaar/wapi/test/playbooks/fail.yml", "cmdline": "ansible-playbook /home/vengaar/wapi/test/playbooks/fail.yml "}