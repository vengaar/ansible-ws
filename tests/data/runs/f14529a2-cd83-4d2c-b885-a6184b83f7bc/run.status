{"pid": 19735, "runid": "f14529a2-cd83-4d2c-b885-a6184b83f7bc", "status": "finished", "state": "failed", "begin": 1551460796.1865423, "end": 1551460797.9021518, "return_code": 2, "description": {"playbook": "/home/vengaar/wapi/test/playbooks/fail.yml", "cmdline": "ansible-playbook /home/vengaar/wapi/test/playbooks/fail.yml "}}