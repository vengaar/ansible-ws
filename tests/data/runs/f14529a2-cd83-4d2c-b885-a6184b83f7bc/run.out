
PLAY [localhost] ***************************************************************

TASK [fail] ********************************************************************
[0;31mfatal: [localhost]: FAILED! => {"changed": false, "msg": "Test playbook fail"}[0m
	to retry, use: --limit @/home/vengaar/wapi/test/playbooks/fail.retry

PLAY RECAP *********************************************************************
[0;31mlocalhost[0m                  : ok=0    changed=0    unreachable=0    [0;31mfailed=1   [0m

